bleachbit/Action.py

```diff
@@ -195,6 +196,21 @@ def get_paths(self):
         for f in itertools.ifilter(self.path_filter, self._get_paths()):
             yield f
 
+    def __str__(self):
+        ret = self.action_key+":   "
+        ret += "and   ".join(self.paths)
```

Comment:
Under Google Chrome I see

`delete:   /home/z/.config/chromium/Safe Browsing Channel IDs-journaland   %LocalAppData%\Chromium\User Data/Safe Browsing Channel IDs-journal`

There is missing whitespace before "and"